Makefile

```diff
@@ -254,41 +238,16 @@ build/src/libultra_code_O1/llcvt.o: src/libultra_code_O1/llcvt.c
 	@$(OBJDUMP) -d $@ > $(@:.o=.s)
 
 assets/%.c: assets/%.xml
-#	$(ZAPD) bsf -i $< -o $(dir $@)
 	$(ZAPD) bsf -eh -i $< -o $(dir $<)
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o build/$(@:.c=.o) $@
 
-build/%.rgba32.inc.c: %.rgba32.png
-	$(ZAPD) btex -tt rgba32 -i $< -o $@
-
-build/%.rgb5a1.inc.c: %.rgb5a1.png
-	$(ZAPD) btex -tt rgb5a1 -i $< -o $@
-
-build/%.i4.inc.c: %.i4.png
-	$(ZAPD) btex -tt i4 -i $< -o $@
-
-build/%.i8.inc.c: %.i8.png
-	$(ZAPD) btex -tt i8 -i $< -o $@
-
-build/%.ia4.inc.c: %.ia4.png
-	$(ZAPD) btex -tt ia4 -i $< -o $@
-
-build/%.ia8.inc.c: %.ia8.png
-	$(ZAPD) btex -tt ia8 -i $< -o $@
-
-build/%.ia16.inc.c: %.ia16.png
-	$(ZAPD) btex -tt ia16 -i $< -o $@
-
-build/%.ci4.inc.c: %.ci4.png
-	$(ZAPD) btex -tt ci4 -i $< -o $@
-
-build/%.ci8.inc.c: %.ci8.png
-	$(ZAPD) btex -tt ci8 -i $< -o $@
+build/%.inc.c: %.png
+	$(ZAPD) btex -eh -tt $(lastword ,$(subst ., ,$(basename $<))) -i $< -o $@
 
 build/assets/%.bin.inc.c: assets/%.bin
-	$(ZAPD) bblb -i $< -o $@
+	$(ZAPD) bblb -eh -i $< -o $@
 
 build/assets/%.jpg.inc.c: assets/%.jpg
-	$(ZAPD) bren -i $< -o $@ -eh
+	$(ZAPD) bren -eh -i $< -o $@ -eh
```

Comment:
I'm not sure what `-eh` does, but is it intended to appear twice in the command here?

Reply:
`-eh` is to enable ZAPD's error handler. And it should appear twice.